.mailmap

```diff
@@ -1674,3 +1674,4 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
 袁野 (Yuan Ye) <yuanyelele@tutanota.com>
+Peter Stahlecker <peter.stahlecker@gmail.com>   Peter230655 <peter.stahlecker@gmail.com>
```

Comment:
```suggestion
Peter Stahlecker <peter.stahlecker@gmail.com>   Peter230655 <83544146+Peter230655@users.noreply.github.com>
```

Reply:
> 

This change is to be made in the .mailmap  ?

Reply:
yes

Reply:
I think worked unless I made a typo.

![image](https://github.com/user-attachments/assets/b09e2657-2160-4535-b29d-c15094081c8c)

this is what I got with cat .mailmap

![image](https://github.com/user-attachments/assets/117c4449-5476-4120-8af5-319ea50b504d)

Is a PR required, or not?

**Thanks for all the help!!**
Before I turn 75, I will know it for sure! :-)